sdk/src/client/bindings/python/examples/build_alias.py

```diff
@@ -8,45 +8,18 @@
     'rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy')
 
 alias_id = '0x0000000000000000000000000000000000000000000000000000000000000000'
-# `hello` hex encoded
-state_metadata = '0x68656c6c6f'
+state_metadata = data='0x'+'Hello, World!'.encode('utf-8').hex()
 unlock_conditions = [
-    # StateControllerAddressUnlockCondition
-    {'type': 4, 'address': {'type': 0, 'pubKeyHash': hexAddress}},
-    # GovernorAddressUnlockCondition
-    {'type': 5, 'address': {'type': 0, 'pubKeyHash': hexAddress}},
+    StateControllerAddressUnlockCondition(Ed25519Address(hexAddress)),
+    GovernorAddressUnlockCondition(Ed25519Address(hexAddress))
 ]
 features = [
-    {
-        # sender feature
-        'type': 0,
-        'address': {
-            'type': 0,
-            'pubKeyHash': hexAddress,
-        },
-    },
-    {
-        # MetadataFeature
-        'type': 2,
-        # `hello` hex encoded
-        'data': '0x68656c6c6f',
-    }
+    SenderFeature(Ed25519Address(hexAddress)),
+    MetadataFeature(data='0x'+'Hello, World!'.encode('utf-8').hex())
 ]
 immutable_features = [
-    {
-        # issuer feature
-        'type': 1,
-        'address': {
-            'type': 0,
-            'pubKeyHash': hexAddress,
-        },
-    },
-    {
-        # MetadataFeature
-        'type': 2,
-        # `hello` hex encoded
-        'data': '0x68656c6c6f',
-    },
+    IssuerFeature(issuer=Ed25519Address(hexAddress)),
```

Comment:
Why has Issuer a named field and not Sender?

Reply:
Hmm. Not sure. Seems like a copy paste error 🤔
Both have named fields.

Reply:
Can you harmonise pls? I also saw Metadata has an explicitly named field, not sure it's necessary

Reply:
Sure. It's on my todo for today. Just need to catch up on my other tasks first 😅

Reply:
Alles gut